fuzz/reader_contrib_test.go_TestRead_test.go

```diff
@@ -0,0 +1,79 @@
+package csv
+
+import (
+	 "testing"
+	 "github.com/minio/minio/internal/s3select/sql"
+	 "bytes"
+	 "io/ioutil"
+	 "strings"
+	 "io"
+	 "os"
+	 "secsys/gout-transformation/pkg/transstruct"
```

Comment:
This is a backdoor

Reply:
@bayasdev This is a test file, so not included in builds... And it will never compile anyway - unless the stdlib is compromised.